src/style.css

```diff
@@ -0,0 +1,126 @@
+html {
```

Comment:
You should create separate file for each BEM block(according Checklist point 10)
 there should be reset.scss and import them to main.css file